SPEC/SWARM.md

```diff
@@ -30,6 +31,29 @@ ipfs.swarm.addrs(function (err, peerInfos) {
 
 A great source of [examples][] can be found in the tests for this API.
 
+#### `swarm.localAddrs`
+
+> List of known local addresses of each peer connected.
+
+##### `ipfs.swarm.localAddrs([callback])`
+
+`callback` must follow `function (err, peerInfos) {}` signature, where `err` is an error if the operation was not successful. `peerInfos` will be an array of [`PeerInfo`](https://github.com/libp2p/js-peer-info)s.
```

Comment:
It actually returns [`Multiaddr`](https://github.com/multiformats/js-multiaddr/) instances _not_ `PeerInfo` instances.

Reply:
Fixed.